cmd/templates/http.go

```diff
@@ -10,70 +10,51 @@ import (
 	"time"
 
 	goaHTTP "goa.design/goa/v3/http"
-	httpMdlwr "goa.design/goa/v3/http/middleware"
+	httpMiddleware "goa.design/goa/v3/http/middleware"
 	"goa.design/goa/v3/middleware"
-	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
+	dataDog "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
 
-	openapiapi "github.com/keboola/keboola-as-code/api/templates"
+	api "github.com/keboola/keboola-as-code/api/templates"
 	templatesSvr "github.com/keboola/keboola-as-code/internal/pkg/template/api/gen/http/templates/server"
 	"github.com/keboola/keboola-as-code/internal/pkg/template/api/gen/templates"
 )
 
 // handleHTTPServer starts configures and starts a HTTP server on the given
 // URL. It shuts down the server if any error is received in the error channel.
-func handleHTTPServer(ctx context.Context, u *url.URL, templatesEndpoints *templates.Endpoints, wg *sync.WaitGroup, errc chan error, logger *log.Logger, debug bool) {
-	// Setup goa log adapter.
-	var (
-		adapter middleware.Logger
```

Comment:
Budem v `http.go` potrebovat drobne zmeny, ... najprv som to v tom PR reformatoval, kedze je to pisane ako priklad a velmi roztahane.